src/components/CorporateCampaign/CampaignPartner.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<section class="campaign-partner section">
+		<div class="row">
+			<div class="small-12 medium-10 large-6 columns">
+				<img
+					v-if="heroLogo.url"
+					class="campaign-partner__img"
+					:src="heroLogo.url"
+					:alt="heroLogo.title"
+				>
+			</div>
+			<div class="small-10 large-6 columns">
+				<h2
+					v-if="headline"
+					class="campaign-partner__header"
+				>
+					{{ headline }}
+				</h2>
+				<div
+					v-if="bodyCopy"
+					class="campaign-partner__body"
+					v-html="bodyCopy"
+					ref="partnerBodyCopy"
+				></div>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+export default {
+	props: {
+		partnerAreaContent: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	computed: {
+		heroLogo() {
```

Comment:
a little confusing to have this called hero...but no biggie.

Reply:
Good catch, agreed